include/experimental/kernel/gemm/impl/int4_dequantize_kslicing_xe.hpp

```diff
@@ -95,7 +95,7 @@ class gemm_universal_t<
   using dtype_c = typename mem_desc_c_t::dtype;
   using dtype_scale = typename mem_desc_scale_t::dtype;
   using dtype_zero_pt = typename mem_desc_zero_pt_t::dtype;
-  using matAcc_t = typename gemm_t::matAcc_t;
+  using matAcc_t = typename gemm_t::matC_t;
```

Comment:
I think this is confusing

Reply:
Here, a matacc_t is added to each mirco kernel to cache the intermediate results of for k-dim. The output of the mirco kernel is renamed to matC, and in kslicing, matc is a cache tile. In fact, both matC and matacc are cache tiles.

Reply:
matC_t and matAcc_t may have different data type ?

Reply:
both mma_acc_type

Reply:
If look other place like fp16 x fp16 to got fp16, generally, dtype_c will be fp16 while dtype_acc will be float to keep the precision as possible, is that the int4 not need things like this ?